### DISCUSSION TITLE
[Closed] FlexBasis Extension method overload + docs

### DISCUSSION THREAD

**Discussion by danielftz** [CommunityToolkit/Maui.Markup#34] [New Feature Discussions]
I would like to propose an overload method for Basis<TBindable>

```cs
	public static TBindable Basis<TBindable>(this TBindable bindable, float length, bool isRelative = false) where TBindable : BindableObject
	{
		return bindable.Basis(new FlexBasis(length, isRelative));
	}
```

I can also fill out the docs and sample for all FlexLayout Extension methods if needed.

**Comment by TheCodeTraveler** [comment]
Thanks @danielftz! This looks good to me, but I'm admittedly not an expert with `FlexLayout`! 

I'll ping the team to see if anyone with more experience than me has any recommendations before we open up a `Proposal` 👍

**Reply by bijington** [reply]
I don't know much but given it maps to existing .NET MAUI methods it sounds fine to me

**Reply by TheCodeTraveler** [reply]
Thanks Shaun! 

@danielftz Feel free to open a Proposal for this! I'd be happy to Champion it 👍 

Just FYI - we will be releasing our first Release Candidate alongside .NET MAUI's first RC in ~2 weeks, after which we will implement a feature freeze.

If you're able to open the Proposal and the associated PR within the next two weeks, then we'll be able to add this feature in time for the RC! If not, no worries, we can add it after we lift our feature freeze in May.

**Comment by TheCodeTraveler** [comment]
Promoted to Proposal: https://github.com/CommunityToolkit/Maui.Markup/issues/48

**Comment by CliffAgius** [comment]
Closed and answered
